### ISSUE TITLE
[Android] Webview was very slow in flutter SDK 3.27.x

### ISSUE THREAD

**Issue by ttb-inc** [pichillilorenzo/flutter_inappwebview#2468]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Opening and closing a webview in a new window using the navigator was much slower than before.

Pages created with widgets usually don't slow down, but only pages using webviews are exceptionally slow.

This was not the case with the previous flutter SDK 3.24.5 (downgrade test).

I'd love to hear about other people's experiences and tests.
Thanks a lot.

### Expected Behavior

You want normal behavior.

### Steps with code example to reproduce

<details>
<summary>Steps with code example to reproduce</summary>

```dart
step1 : navigator.push(webview0);
step2 : navigator.push(webview1); in webview0
```
</details>

### Stacktrace/Logs

...

### Flutter version

v3.27.0, v3.27.1

### Operating System, Device-specific and/or Tool

Android 34+, Android 31, Android 29, 

### Plugin version

v6.1.5

### Self grab

- [x] I'm ready to work on this issue!

**Comment by cooaer** [comment]
same problem

**Comment by mugglecoder** [comment]
same problem

**Comment by cooaer** [comment]
@ttb-inc Set `useHybridComposition:false`  can solve this problem, but I don't know if it will cause new problems.

**Comment by ttb-inc** [comment]
> @ttb-inc Set `useHybridComposition:false` can solve this problem, but I don't know if it will cause new problems.

I'll test it right now.
Thank you.

**Comment by ttb-inc** [comment]
> @ttb-inc Set `useHybridComposition:false` can solve this problem, but I don't know if it will cause new problems.

With "useHtbridComposition:false", there seems to be some flickering on page transitions.
I'll have to test this a bit more.
Thanks a lot.

**Comment by natsuk4ze** [comment]
I solved this issue by disabling Impeller on Android.

Add this line in `AndroidManifest.xml`.
```AndroidManifest.xml
        <meta-data
            android:name="io.flutter.embedding.android.EnableImpeller"
            android:value="false" />
```

**Comment by METASTARIncorporated** [comment]
> I solved this issue by disabling Impeller on Android.
> 
> Add this line in `AndroidManifest.xml`.
> 
>         <meta-data
>             android:name="io.flutter.embedding.android.EnableImpeller"
>             android:value="false" />


I tested it.
It works fine.
Let's test further to make sure there are no other issues.
Thanks a lot.

**Comment by natsuk4ze** [comment]
Yes, with flutter 3.27, Android Impeller is now opted in by default, but it still seems unstable.
